rs/config/src/embedders.rs

```diff
@@ -42,18 +42,18 @@ const DEFAULT_WASMTIME_RAYON_COMPILATION_THREADS: usize = 10;
 /// The number of rayon threads use for the parallel page copying optimization.
 const DEFAULT_PAGE_ALLOCATOR_THREADS: usize = 8;
 
-/// Sandbox process eviction does not activate if the number of sandbox
-/// processes is below this threshold.
-pub(crate) const DEFAULT_MIN_SANDBOX_COUNT: usize = 500;
-
 /// Sandbox process eviction ensures that the number of sandbox processes is
 /// always below this threshold.
-pub(crate) const DEFAULT_MAX_SANDBOX_COUNT: usize = 2_000;
+pub(crate) const DEFAULT_MAX_SANDBOX_COUNT: usize = 5_000;
 
 /// A sandbox process may be evicted after it has been idle for this
 /// duration and sandbox process eviction is activated.
 pub(crate) const DEFAULT_MAX_SANDBOX_IDLE_TIME: Duration = Duration::from_secs(30 * 60);
 
+/// Sandbox processes may be evicted if their total RSS memory size
+/// exceeds 100 GiB.
+pub(crate) const DEFAULT_MAX_SANDBOXES_RSS_SIZE: NumBytes = NumBytes::new(100 * 1024 * 1024 * 1024);
```

Comment:
Are you sure we can afford 100 GB worth of sandboxes?

Reply:
That's the current state of the mainnet (2k sandboxes times 50 MiB each). Metrics indicate that nodes have a minimum of 270+ GiB of available memory, with up to 2k of active sandboxes.

As discussed offline, we should prioritize work to mitigate risks.

Reply:
I don't think this is the way it works; we had that document with max memory usage per component for a reason; we're all of a sudden doubling something without taking anything back.

Reply:
Discussed offline. The document is outdated, as two weeks ago we've doubled the number of sandboxes to 2k on mainnet (up to 50 MiB each totaling 100 GiB). Anyway, reduced the limit to 50 GiB.